recipes/conversation_view/feed_pill/feed_item_pill.test.js

```diff
@@ -1,133 +1,117 @@
-    /*
-     * Test(s) to ensure that the component is correctly rendering
-     */
-
     describe('Default render', function () {
       it('should exist', () => {
         expect(wrapper.exists()).toBe(true);
       });
       it('should render a feed item pill', () => {
+        console.log(wrapper.html());
         expect(feedItemPill.exists()).toBeTruthy();
         expect(icon.exists()).toBe(true);
         expect(hoverIcon.exists()).toBe(false);
-        expect(contentSlot.exists()).toBe(false);
-      });
-    });
-
-    describe('Expand true', function () {
-      beforeAll(() => {
-        propsData = {
-          ...basePropsData,
-          expanded: true,
-        };
-      });
-      it('should show contentSlot', () => {
-        expect(contentSlot.exists()).toBe(true);
+        expect(wrapper.find('[data-qa="content-element"]').exists()).toBe(false);
       });
     });
   });
 
   describe('Accessibility Tests', function () {
-    /*
-     * Test(s) to ensure that the component is accessible
-     */
-
     describe('Default Render', function () {
       it('should render a reaction button', () => {
-        expect(feedItemPill.attributes('aria-label')).toBe(basePropsData.ariaLabel);
+        expect(feedItemPill.attributes('aria-label')).toBe(MOCK_ARIA_LABEL);
       });
     });
   });
 
   describe('Interactivity Tests', function () {
-    /*
-     * Test(s) to ensure that the component correctly handles user input
-     */
-
-    describe('Click Feed Item Pill event', async function () {
+    describe('Toggle feed item pill', function () {
       it('Should emit a click event', async () => {
         await feedItemPill.trigger('click');
-        itBehavesLikeEmitsExpectedEvent(wrapper, 'click');
+        await wrapper.vm.$nextTick();
+        expect(wrapper.find('[data-qa="content-element"]').exists()).toBe(true);
       });
-    });
 
-    // Why this no work with trigger focus or trigger mouseenter
-    describe.skip('Hover Feed Item Pill event', async function () {
-      beforeAll(() => {
-        feedItemPill.trigger('focus');
+      describe('toggleable false', function () {
+        beforeAll(() => {
+          mockProps = {
+            toggleable: false,
+          };
+        });
+
+        it('should exist', () => {
+          expect(wrapper.exists()).toBe(true);
+        });
+        it('should not respond to clicks', () => {
+          expect(feedItemPill.exists()).toBeTruthy();
+          expect(icon.exists()).toBe(true);
+          expect(hoverIcon.exists()).toBe(false);
+          expect(wrapper.find('[data-qa="content-element"]').exists()).toBe(false);
+        });
       });
+    });
+
+    // Test not working
+    describe.skip('Hover Feed Item Pill event', function () {
       it('should show a different icon', async () => {
+        await feedItemPill.trigger('focus');
+        _setChildWrappers();
         expect(icon.exists()).toBe(false);
         expect(hoverIcon.exists()).toBe(true);
       });
```

Comment:
Maybe you can emit an event on focus and check if the event has been fired?

IDK what else to suggest here, any ideas? @braddialpad @iropolo

Reply:
I've always had big problems with jsdom and setting focus.. 1 sec taking a look